poetry.lock

```diff
@@ -43,8 +43,8 @@ python-versions = "*"
 name = "apeye"
 version = "1.1.0"
 description = "Handy tools for working with URLs and APIs."
-category = "dev"
-optional = false
+category = "main"
```

Comment:
Ok, so I am not used to `poetry`, all I did was run `poetry lock --no-update`.

Let me know if I messed something up there!

Maybe we should something to the `CONTRIBUTING.md` regarding poetry and the lock file?